include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -4994,17 +4993,15 @@ FEEvaluationBase<dim, n_components_, Number, is_face, VectorizedArrayType>::
             for (unsigned int e = 0; e < dim; ++e)
               hessian_out[comp][d][d] +=
                 jac_grad[d][e] *
-                this->gradients_from_hessians_quad[(comp * dim + e) * nqp +
-                                                   q_point];
+                this->gradients_quad[(comp * dim + e) * nqp + q_point];
```

Comment:
Does this change make sense? `gradients_from_hessians_quad` -> `gradients_quad`. As far as I remember @bergbauer told me that a second storage is needed for Hessians.

Reply:
Yes, but we only need the storage for the multiplication by test functions, not the evaluation (where we can use the `gradients_quad` field directly, as that field is not overwritten anywhere). This approach avoids a rather expensive copy operation in terms of cache (unless one also submits the hessians, which is not as widespread as the get part).